express/pipeline_utils.py

```diff
@@ -12,6 +12,31 @@
 logger = logging.getLogger(__name__)
 
 
+def get_kubeflow_type(python_type: str):
```

Comment:
This should probably not be in `pipeline_utils.py` if we're using it in the component.